CUDADataFormats/Track/interface/PixelTrackUtilities.h

```diff
@@ -0,0 +1,241 @@
+#ifndef CUDADataFormats_Track_PixelTrackUtilities_h
+#define CUDADataFormats_Track_PixelTrackUtilities_h
+
+#include <Eigen/Dense>
+#include <Eigen/Core>
+#include "Geometry/CommonTopologies/interface/SimplePixelTopology.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/HistoContainer.h"
+#include "DataFormats/SoATemplate/interface/SoALayout.h"
+
+namespace pixelTrackSoA {
```

Comment:
why this is `pixelTrackSoA`, while `CUDADataFormats/Track/interface/TrackSoAHeterogeneousDevice.h` and `TrackSoAHeterogeneousHost.h` use `pixelTrack` ?

Reply:
Apparently no reason, I'm making them uniform.